### ISSUE TITLE
NonceReboot11Cli stopped working with the latest Unc0ver Pre-release

### ISSUE THREAD

**Issue by Nikostito** [pwn20wndstuff/Undecimus#258]
Hello, so after I installed the latest v.2.2.0 beta of Unc0ver the NonceReboot11 cli tweak stopped orking and is giving me this error after I try every command ( -g and -s flasgs): 

`[ERR]: No such file or directory`

Has that something to do with the fact that substitute support was dropped in this version? Can you please look into this? Thank you in advance

**Comment by pwn20wndstuff** [comment]
Yes - I will rewrite my changes to noncereboot11 - They needed a rewrite anyway.

**Comment by pwn20wndstuff** [comment]
@Nikostito You're welcome. https://github.com/pwn20wndstuff/nonceutil
